controller/src/main/java/io/pravega/controller/store/stream/PersistentStreamBase.java

```diff
@@ -918,7 +930,7 @@ private long findSegmentSplitsMerges(List<StreamSegmentRecord> referenceSegments
                             EpochRecord duplicateActiveEpoch = EpochRecord.builder().epoch(committingTxnRecord.getNewActiveEpoch())
                                                                           .referenceEpoch(activeEpochRecord.getReferenceEpoch())
                                                                           .segments(duplicateActiveSegments)
-                                                                          .creationTime(timeStamp).build();
+                                                                          .creationTime(timeStamp + 1).build();
```

Comment:
Why +1?

Reply:
replied above.. duplicateActiveEpoch = txnDuplicateEpoch + 1 
this ensures monotonically increasing..
earlier we used the same time for both.